src/GuiBase/Viewer/Gizmo/Gizmo.hpp

```diff
@@ -41,9 +44,17 @@ namespace Ra
                 }
             }
 
+            // shows or hides the gizmos drawables.
+            void show(bool on)
+            {
+                for (auto ro: m_renderObjects)
+                {
+                    Ra::Engine::RadiumEngine::getInstance()->getRenderObjectManager()->getRenderObject(ro)->setVisible(on);
```

Comment:
I suggest to temporary store the pointer to the RenderObjectManager out of the loop (code readability)